README.md

```diff
@@ -6,69 +6,85 @@ Utilities for interacting with a TPM2.0 used for access to the Xaptum Edge Netwo
 
-# Configure the build
-cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON
+# Install the repository GPG key
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys c615bfaa7fe1b4ca
 
-# Build the library
-cmake --build .
+# Install the library
+apt-get install libxaptum-tpm-dev
 ```
 
-In addition to the standard CMake options the following configuration
-options and variables are supported.
+### Debian Stretch
+
+```bash
+# Add the repository to APT sources
+echo "deb http://dl.bintray.com/xaptum/deb stretch main" | sudo tee -a /etc/apt/sources.list
+
+# Install the repository GPG key
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys c615bfaa7fe1b4ca
 
-### Static vs Shared Libary
-If `BUILD_SHARED_LIBS` is set, the shared library is built. If
-`BUILD_STATIC_LIBS` is set, the static library is built. If both are
-set, both libraries will be built.  If neither is set, the static
-library will be built.
+# Install the library
+apt-get install libxaptum-tpm-dev
+```
 
-### Static Library Name
-`STATIC_SUFFIX`, if defined, will be appended to the static library
-name.  For example,
+### Homebrew (MacOS)
 
 ```bash
-cmake .. -DBUILD_STATIC_LIBS=ON -DSTATIC_SUFFIX=_static
-cmake --build .
-```
+brew tap xaptum/xaptum
 
-will create a static library named `libxaptum-tpm_static.a`.
+brew install xaptum-tpm
+```
 
-### Force Position Independent Code (-fPIC)
-Set the standard CMake variable `CMAKE_POSITION_INDEPENDENT_CODE` to
-`ON` to force compilation with `-fPIC` for static libraries.  The
-default is `OFF` for static libraries and `ON` for shared libraries.
+## Installation from Source
 
-### Disable Building of Tests
-Set the standard CMake variable `BUILD_TESTING` to `OFF` to disable
-the building of tests.  The default value is `ON`.
+### Build Dependencies
 
-## Installation
+* CMake (version 3.0 or higher)
+* A C99-compliant compiler
+* A POSIX-compliant platform
 
-CMake creates a target for installation.
+### Building
 
 ```bash
+# Create a subdirectory to hold the build
+mkdir -p build
 cd build
-cmake --build . --target install
+
+# Configure the build
+cmake .. -DCMAKE_BUILD_TYPE=Release
+
+# Build the library
+cmake --build .
 ```
 
-Set the `CMAKE_INSTALL_PREFIX` variable when configuring the build to
-modify the installation location.
+### CMake Options
+
+The following CMake configuration options are supported.
+
+| Option                          | Values         | Default | Description                              |
+|---------------------------------|----------------|---------|------------------------------------------|
+| CMAKE_BUILD_TYPE                | Release        |         |                                          |
+|                                 | Debug          |         |                                          |
+|                                 | RelWithDebInfo |         |                                          |
```

Comment:
We also support `RelWithSanitize`, which enables the address- and undefined-behavior-sanitizer options in the compiler, to include run-time checks in the code.

Should we document that?